restonomer-docs/source/restonomer_transformations.md

```diff
@@ -282,3 +282,56 @@ The transformed response will now have the columns with the desired data types:
   "col_B": "[{'ZipCodeType':'STANDARD','Zipcode':704}]"
 }
 ```
+## ReplaceStringInColumnValue
+
+It lets the user cast the data type of multiple columns to the desired different types at once.
+
+This transformation expects user to provide below inputs:
+
+| Input Arguments | Mandatory | Default Value | Description                          |
+|:----------------|:---------:|:-------------:|:-------------------------------------|
+| column-Name     |    Yes    |       -       | It is the column name                |
```

Comment:
Make N to small letter. It should be "column-name"

Reply:
Done